mozsprint2018DataProcessingSentinel.md

```diff
@@ -0,0 +1,114 @@
+# Suggested workflow for using satellite data to classify objects for mapping
+
+## Mozsprint 2018 
+
+### Hosted in University of Manchester, Allan Turning building by Rachael Ainsworth
+
+*Contrubutors*
+
+* Sophie Ashcroft - ms.sophie.ashcroft@gmail.com
+* Rohini Joshi - rohini.joshi@manchester.ac.uk
+* Elliott Polzin - elliott.polzin3@gmail.com
+* Robert Lyon - robert.lyon@manchester.ac.uk
+* Therese Cantwell - therese.cantwell@manchester.ac.uk
+* Joe Fennell - joseph.fennell@manchester.ac.uk
+
+*Document target user*
+
+This is written for individuals confindent and compitent in Machine Learning technologies and systems.
+
+### Document Goal
+
+After reading into the context and scope of the Crowd2Map project we found that with the time and human constraits we are under, the best way for us to contribute would be to create a set of recommendations for how in the future contributors can automate mapping by using a combination of satallite and data processing technologies.
+
+Below is a proposed workflow of what data should be collected, from where, how to train a K-means algorithm on it and what this proposed solution will deliver. Finally there is further information on how this process has worked in practise on a map of Peurto Rico.
+
```

Comment:
What does K-means mean please? Does anyone have any links to mapping examples?

Reply:
https://en.wikipedia.org/wiki/K-means_clustering